recipe/meta.yaml

```diff
@@ -40,6 +40,7 @@ requirements:
     - {{ compiler('cxx') }}
     - pkg-config  # [not win]
```

Comment:
you know that pkg-config is available on Windows too?

Reply:
I didn't. I didn't test this for anything but linux though. The "mac-volatile..." patch does not successfully apply so those builds fail. And on s390x the build fails because boost is not available as a dependency (boost would now be a dependency unless a little more work is done to configure cmake to build without the boost dependent splash functionality).

Reply:
the mac-volatile patch is a giant hack to work-a-round a clang issue on casting of volatiles.  I hope we won't need it for newer version anymore, so it would be good if it got no longer required